hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/recovery/DBRMStateStore.java

```diff
@@ -446,12 +452,25 @@ public Object performTask() throws StorageException {
         ApplicationState state = new ApplicationState(appIdString, appState,
                 user, name, stateN);
         DA.add(state);
+        logProvenance(state, submitTime, startTime, finishTime);
```

Comment:
It does not seems very related to the remaining of the pull request to me. Do we need this modification?

Reply:
Ok, sorry I thought it was in tests, but it is just in a following module. Sorry.